corehq/apps/hqadmin/templates/hqadmin/partials/deploy_history.html

```diff
@@ -12,56 +12,25 @@ <h2 class="panel-title">
             <tr>
                 <th>Date</th>
                 <th>User</th>
-                <th>Branch</th>
-                <th>Last Commit</th>
-                <th></th>
+                <th>Diff url</th>
             </tr>
             </thead>
             <tbody>
             {% for deploy in deploy_history %}
-                {% with deploy.code_snapshot.commits|first as commit%}
                 <tr>
                     <td>
                         {{ deploy.date|naturaltime }}<br/>
                         <small>{{ deploy.date }}</small>
                     </td>
                     <td>{{ deploy.user }}</td>
-                    <td>{{ deploy.code_snapshot.current_branch }}</td>
                     <td>
-                        {% include "hqadmin/partials/commit.html" %}
```

Comment:
@czue i feel like you might have some opinions about deleting this. we don't save the last commit anymore so we won't have this column. i've never found it that particularly useful because the last commit is usually just submodule updates and the diff url is usually more relevant. the other `Details` modal that i deleted below seems to be completely broken so I figured no one would complain if i deleted that